lib/i18n/js.rb

```diff
@@ -149,10 +149,13 @@ def self.filter(translations, scopes)
 
     # Initialize and return translations
     def self.translations
-      ::I18n.backend.instance_eval do
+      backend_translations = ::I18n.backend.instance_eval do
         init_translations unless initialized?
-        translations.slice(*::I18n.available_locales)
+        translations
       end
+
+      return backend_translations if available_locales.empty?
```

Comment:
Why do we want to return all backend translations when `available_locales` is empty?
We should remind them or at least return nothing (so developers notice they might have set something wrong)